shortex.tex

```diff
@@ -1,188 +1,622 @@
+we might want to use commands like
+\verb!\fkh...!  instead of \verb!\f[kh]...!.
+We can accomplish this using the \verb!\parsefontstylestrings! command,
+with syntax
+\begin{verbatim}
+\parsefontstylestrings{{<fstr1>}{<fstr2}...}{<alphabet>}
+\end{verbatim}
+For example, to define ``bold hat'' and ``caligraphic hat'' styles
+for the characters A, B, C, and D, we would use the command 
+\begin{verbatim}
+\parsefontstylestrings{{kh}{ch}}{ABCD}
+\end{verbatim}
+\parsefontstylestrings{{kh}{ch}}{ABCD}
+and then in the \LaTeX~document, use the commands
+\verb!\fkhA \fkhB \fkhC \fkhD! and
+\verb!\fchA \fchB \fchC \fchD! 
+to obtain the following symbols:
+\[
+\fkhA \fkhB \fkhC \fkhD 
+\fchA \fchB \fchC \fchD 
+\]
+As another example, for ``bold hat'' applied to $\alpha$, $\beta$, and $\gamma$, we would use the syntax
+\begin{verbatim}
+\parsefontstylestrings{{kh}}{\alpha\beta\gamma}
+\end{verbatim}
+\parsefontstylestrings{{kh}}{\alpha\beta\gamma}
+and then in the \LaTeX~document, use the commands
+\verb!\fkhalpha \fkhbeta \fkhgamma!
+to obtain the following symbols:
+\[
+	\fkhalpha \fkhbeta \fkhgamma
+\]
+
+For convenience we also provide a few common alphabets of symbols 
+for use in the \verb!\parsefontstylestrings! command.
+Note that not every Greek character has an uppercase version (in cases where it is
+identical to its Roman counterpart).
+
+\bcent
+\btabr{@{}lll@{}}
+\toprule
+Syntax & Characters  \\ \midrule
+\verb!\lowercaseRoman! & abcdefghijklmnopqrstuvwxyz \\
+\verb!\uppercaseRoman! & ABCDEFGHIJKLMNOPQRSTUVWXYZ \\
+\verb!\lowercaseGreek! & alpha,beta,gamma,delta,epsilon,zeta,eta,theta\\
+& iota,kappa,lambda,mu,nu,xi,omicron,pi,rho\\
+& sigma,tau,upsilon,phi,chi,psi,omega\\
+\verb!\uppercaseGreek! & Gamma,Delta,Theta,Lambda,Xi,Pi,Sigma\\
+& Upsilon,Phi,Psi,Omega \\
+\bottomrule
+\etabr
+\ecent
+
+\subsection{Greek characters and variants}
+
+ShorTeX defines a number of shorthands for Greek characters and variants.
+
+\bcent
+\btabr{@{}lll@{}}
+\toprule
+Letter & Syntax & Symbol  \\ \midrule
+epsilon	& \verb!\eps! & $\eps$ \\
+upsilon	& \verb!\ups! & $\ups$ \\
+variant epsilon	& \verb!\veps! & $\veps$ \\
+variant theta	& \verb!\vtheta! & $\vtheta$ \\
+variant pi	& \verb!\vpi! & $\vpi$ \\
+variant rho	& \verb!\vrho! & $\vrho$ \\
+variant sigma	& \verb!\vsigma! & $\vsigma$ \\
+variant phi	& \verb!\vphi! & $\vphi$ \\
+variant kappa	& \verb!\vkappa! & $\vkappa$ \\
+\bottomrule
+\etabr
+\ecent
+
+\subsection{Referencing figures, equations, tables, etc.}
+
+ShorTeX includes the \texttt{cleveref} package, which simplifies the process
+of typesetting references. Use the \verb!\cref! command to automatically
+typeset the names of the objects you reference (including properly handling multiple references). 
+For example, if \verb!\label{fig:first}! is applied to the first figure in the document,
+\begin{verbatim}
+In \cref{fig:first}, you can see...
+\end{verbatim}
+would typeset as ``In Figure 1, you can see...''
```

Comment:
I think this would show up as "In Fig. 1". To get "In Figure 1" you would need to use `\Cref` to indicate it's at the beginning of a sentence

Reply:
good pt -- fixed